src/Libraries/Builtin/Dictionary.cs

```diff
@@ -55,29 +52,20 @@ private Dictionary(ImmutableDictionary<string, object> dict)
             ///     Produces the keys in a Dictionary.
             /// </summary>
             /// <returns name="keys">The keys of the Dictionary</returns>
-            public IEnumerable<string> Keys
-            {
-                get { return D.Keys; }
-            }
+            public IEnumerable<string> Keys => D.Keys;
```

Comment:
@QilongTang guess it's time for us to start using c#6 features in revit branches.